packages/twenty-front/src/modules/object-record/record-filter/utils/turnObjectDropdownFilterIntoQueryFilter.ts

```diff
@@ -24,6 +24,128 @@ export type ObjectDropdownFilter = Omit<Filter, 'definition'> & {
   };
 };
 
+const applyEmptyFilters = (
+  operand: ViewFilterOperand,
+  correspondingField: Pick<Field, 'id' | 'name'>,
+  objectRecordFilters: RecordGqlOperationFilter[],
+  filterType:
+    | 'URL'
+    | 'Address'
+    | 'String'
+    | 'Number'
+    | 'Date'
+    | 'Relation'
+    | 'CompositeString' = 'String',
+) => {
+  let emptyRecordFilter: RecordGqlOperationFilter = {};
+
+  switch (filterType) {
+    case 'String':
+      emptyRecordFilter = {
+        or: [
+          { [correspondingField.name]: { ilike: '' } as StringFilter },
+          { [correspondingField.name]: { is: 'NULL' } as StringFilter },
+        ],
+      };
+      break;
+    case 'CompositeString':
+      // eslint-disable-next-line no-case-declarations
```

Comment:
Use brackets instead `{ ... }`

Reply:
prefer not declare fullNameFilters instead of adding the eslint disable comment
```suggestion
      emptyRecordFilter = {
        and: generateILikeFiltersForCompositeFields(
          '',
          correspondingField.name,
          ['firstName', 'lastName'],
          true,
        ),
      };
```